src/recensio/plone/behaviors/article.py

```diff
@@ -2,6 +2,7 @@
 from plone.autoform.interfaces import IFormFieldProvider
 from plone.dexterity.interfaces import IDexterityContent
 from plone.supermodel import model
+from plone.supermodel.directives import fieldset
```

Comment:
```
from plone.supermodel import model
```
then use `model.fieldset` please.

Reply:
@ale-rt why not directly from `directives`? the imports in plone.supermodel.model are deferred imports.

Reply:
That's the documented way https://github.com/plone/plone.supermodel/blob/master/plone/supermodel/directives.rst#fieldset-support

Reply:
:+1: tnx